### ISSUE TITLE
Cannot Shift+Tab back to editor tab name

### ISSUE THREAD

**Issue by chrmarti** [Microsoft/vscode#21517]
- VSCode Version: Code - Insiders 1.10.0-insider (40fecfaa77f01672e8a147be71f6f7ccb9cf8a12, 2017-02-27T14:09:29.088Z)
- OS Version: Darwin x64 15.6.0
- Extensions: none

1. Tab forward through editor tabs with Voice Over
2. Notice how editor titles and close buttons are read out
3. Shift+Tab backward through editor tabs
4. Notice only the close buttons are read out

**Comment by bpasero** [comment]
@chrmarti to be clear you can tab backwards, it's just voice over not reading out the tab label right?

**Comment by bpasero** [comment]
I can reproduce, weird!

**Comment by chrmarti** [comment]
Correct, it's about Voice Over, tabbing back seems to work otherwise.

**Comment by chirag64** [comment]
So I was able to reproduce this and it seems having `tabindex="0"` with `role="presentation"` might seem to be causing this issue.

I was able to fix this using the integrated Developer tools by removing `tabindex="0"` from the outer div (highlighted in red) and adding `tabindex="0"` to the label div (highlighted in yellow) instead.

I'd like to attempt a PR for this.

Removing the tabindex from outer div seems simple: removing [this line of code](https://github.com/Microsoft/vscode/blob/27a99843dc97a6d6bcf6d18c6912b7873ad826df/src/vs/workbench/browser/parts/editor/tabsTitleControl.ts#L402).

But I'm unable to figure out how to add tabindex to the inner div, some help would be appreciated for this.

Let me know if I'm approaching this in the wrong way.

<img width="676" alt="screen shot 2017-03-19 at 10 14 11 pm" src="https://cloud.githubusercontent.com/assets/591424/24082810/6c14a994-0cf1-11e7-877f-27e587aced9b.png">

**Comment by bpasero** [comment]
Seems fixed with 1.12.
